### DISCUSSION TITLE
How to maximize width on q-dialog.

### DISCUSSION THREAD

**Discussion by b0ot** [quasarframework/quasar#16117] [General - Components / Directives / etc]
I would like to conditionally maximize the width on the q-dialog component. 
For my use-case, maximizing the component causes it look bad.

From what I could tell, quasar controls this state with: q-dialog__inner--minimized padding.

I would like to set this to 0px when the screen is smaller than 600px in width. 

Everything I have tried in overriding the variables doesn't seem to work for me. 

What is the recommended way to accomplish this?

**Comment by metalsadman** [comment]
try to make a codepen of what you've tried.

**Comment by b0ot** [comment]
Well thanks to coder1337 in the discord I have a working solution:
https://codesandbox.io/p/sandbox/kind-haze-572c28?file=%2Fsrc%2Fpages%2FIndex.vue%3A41%2C1-42%2C22

I just don't get why I needed to change the css property on `q-dialog__inner` and not `q-dialog__inner--minimized`.

When I looked at Chrome in debugging it looked like the computed padding was because of:
`q-dialog__inner--minimized`

![image](https://github.com/quasarframework/quasar/assets/9140903/be64332e-4318-4c4a-8666-d01a02ce1e6b)

![image](https://github.com/quasarframework/quasar/assets/9140903/c56a6417-08a4-4876-aeaf-a17dffec5392)

When I unchecked that box, the padding would disappear.

I even thought I confirmed by looking at the quasar saas source:
```
.q-dialog
  &__title
    font-size: 1.25rem
    font-weight: 500
    line-height: 2rem
    letter-spacing: .0125em
  &__progress
    font-size: 4rem
  &__inner
    outline: 0
    > div
      pointer-events: all
      overflow: auto
      -webkit-overflow-scrolling: touch
      will-change: scroll-position
      border-radius: $generic-border-radius
      box-shadow: $shadow-4
    &--square > div
      border-radius: 0 !important
    > .q-card
      > .q-card__actions .q-btn--rectangle
        min-width: 64px
    &--minimized
      padding: 24px
      > div
        max-height: calc(100vh - 48px)
    &--maximized
      > div
        height: 100%
        width: 100%
        max-height: 100vh
        max-width: 100vw
        border-radius: 0 !important
        top: 0 !important
        left: 0 !important
    &--top, &--bottom
      padding-top: 0 !important
      padding-bottom: 0 !important
    &--right, &--left
      padding-right: 0 !important
      padding-left: 0 !important
    &--left, &--top
      &:not(.q-dialog__inner--animating) > div
        border-top-left-radius: 0
    &--right, &--top
      &:not(.q-dialog__inner--animating) > div
        border-top-right-radius: 0
    &--left, &--bottom
      &:not(.q-dialog__inner--animating) > div
        border-bottom-left-radius: 0
    &--right, &--bottom
      &:not(.q-dialog__inner--animating) > div
        border-bottom-right-radius: 0
    &--fullwidth
      > div
        width: 100% !important
        max-width: 100% !important
    &--fullheight
      > div
        height: 100% !important
        max-height: 100% !important
```

How can I actually see what class is responsible for the padding if not just trial and error?
